server.js

```diff
@@ -7,7 +7,9 @@ var less = require("less");
 
 var app = express();
 
-app.use(express.logger());
+if (require.main === module) {
+    app.use(express.logger());
+}
```

Comment:
I had very much hoped that there would be something like [Python's `__name__ == "__main__"` check](http://stackoverflow.com/q/419163/1468366) and was very glad when I found [this](https://nodejs.org/api/modules.html#modules_accessing_the_main_module) in the Node.js docs.

By the way, this is the same approach I'd like to take in Python when merging all the metrics manipulation modules into a single process: have each file be both a module and an executable script.